mmcls/models/backbones/levit.py

```diff
@@ -0,0 +1,545 @@
+            dilation=1,
+            groups=1,
+            bn_weight_init=1,
+            conv_cfg=None,
+            norm_cfg=dict(type='BN'),
+    ):
+        super(ConvalutionBatchNorm, self).__init__()
+        self.conv_cfg = conv_cfg
+        self.norm_cfg = norm_cfg
+        self.input_channel = in_channel
+        self.output_channel = out_channel
+        _, bn = build_norm_layer(norm_cfg, out_channel)
+        self.bn_weight_init = bn_weight_init
+        self.conv = build_conv_layer(
+            conv_cfg,
+            in_channel,
+            out_channel,
+            kernel_size=kernel_size,
+            stride=stride,
+            padding=pad,
+            dilation=dilation,
+            groups=groups,
+            bias=False)
+        self.bn = bn
+
+    def init_weights(self):
+        super(ConvalutionBatchNorm, self).init_weights()
+        for m in self.modules():
+            if isinstance(m, nn.BatchNorm2d):
+                torch.nn.init.constant_(m.weight, self.bn_weight_init)
+                torch.nn.init.constant_(m.bias, 0)
+
+    @torch.no_grad()
+    def fuse(self):
+        device = next(self.conv.parameters()).device
+        w = self.bn.weight / (self.bn.running_var + self.bn.eps)**0.5
+        w = self.conv.weight * w[:, None, None, None]
+        b = self.bn.bias - self.bn.running_mean * self.bn.weight / \
+            (self.bn.running_var + self.bn.eps) ** 0.5
+        m = build_conv_layer(
+            self.conv_cfg,
+            w.size(1) * self.conv.groups,
+            w.size(0),
+            w.shape[2:],
+            stride=self.conv.stride,
+            padding=self.conv.padding,
+            dilation=self.conv.dilation,
+            groups=self.conv.groups)
+        m.weight.data.copy_(w)
+        m.bias.data.copy_(b)
+
+        m.to(device)
+        return m
+
+
+class LinearBatchNorm(BaseModule):
+
+    def __init__(self,
+                 in_feature,
+                 out_feature,
+                 bn_weight_init=1,
+                 norm_cfg=dict(type='BN1d')):
+        super(LinearBatchNorm, self).__init__()
+        linear = Linear(in_feature, out_feature, bias=False)
+        _, bn = build_norm_layer(norm_cfg, out_feature)
+        self.bn_weight_init = bn_weight_init
+        self.linear = linear
+        self.bn = bn
+
+    def init_weights(self):
+        super(LinearBatchNorm, self).init_weights()
+        for m in self.modules():
+            if isinstance(m, nn.BatchNorm2d):
+                torch.nn.init.constant_(m.weight, self.bn_weight_init)
+                torch.nn.init.constant_(m.bias, 0)
+
+    @torch.no_grad()
+    def fuse(self):
+        device = next(self.linear.parameters()).device
+        w = self.bn.weight / (self.bn.running_var + self.bn.eps)**0.5
+        w = self.linear.weight * w[:, None]
+        b = self.bn.bias - self.bn.running_mean * self.bn.weight / \
+            (self.bn.running_var + self.bn.eps) ** 0.5
+        m = Linear(w.size(1), w.size(0))
```

Comment:
Why do we need to instantiate a new object here instead of directly replacing the original layer's parameters?

Reply:
Because self. linear. bias=None, so need to instantiate a new object.